src/vm/excep.cpp

```diff
@@ -7314,8 +7318,8 @@ AdjustContextForWriteBarrier(
 {
     WRAPPER_NO_CONTRACT;
 
+#ifndef WIN64EXCEPTIONS
 #ifdef _TARGET_X86_
```

Comment:
This ifdef is not really about WIN64EXCEPTIONS. It is about whether OS unwinder is available. Changing it to `#if defined(_TARGET_X86_) && !defined(PLATFORM_UNIX)` maybe more appropriate.